qcodes/instrument_drivers/Minicircuits/USB_SPDT.py

```diff
@@ -0,0 +1,88 @@
+from qcodes import Instrument
+from qcodes.utils import validators as vals
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+
+
+class SwitchChannelUSB(InstrumentChannel):
+    def __init__(self, parent, name, channel_letter):
+        """
+        Args:
+            parent (Instrument): The instrument the channel is a part of
+            name (str): the name of the channel
+            channel_letter (str): channel letter ['a', 'b', 'c' or 'd'])
+        """
+
+        super().__init__(parent, name)
+        self.channel_letter = channel_letter.upper()
+        _chanlist = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
+        self.channel_number = _chanlist.index(channel_letter)
+
+        self.add_parameter('switch',
+                           label='switch',
```

Comment:
can we change label to 'switch {}'.self.channel_letter?
This will make the metadata a lot more readable.

Reply:
I think that is a good Idea